server/src/main/java/com/objectcomputing/checkins/services/reviews/ReviewAssignmentServicesImpl.java

```diff
@@ -47,6 +45,29 @@ public ReviewAssignment save(ReviewAssignment reviewAssignment) {
         return newAssignment;
     }
 
+    @Override
+    public List<ReviewAssignment> saveAll(UUID reviewPeriodId, List<ReviewAssignment> reviewAssignments, Boolean deleteExisting) {
+
+        if(Boolean.TRUE.equals(deleteExisting)) {
```

Comment:
I'm curious about this syntax. Would it change the result if this line were replaced by the following?

```
if (deleteExisting) {
```